cmd/query/app/querysvc/adjuster/resourceattributes_test.go

```diff
@@ -101,13 +102,13 @@ func TestResourceAttributesAdjuster_SpanWithConflictingLibraryAttributes(t *test
 	require.True(t, ok)
 	require.Equal(t, "Java", val.Str())
 
-	val, ok = resultSpanAttributes.Get("jaeger.adjuster.warning")
+	val, ok = resultSpanAttributes.Get(jptrace.WarningsAttribute)
```

Comment:
nit: we should have GetWarnings() method instead (and the actual tag name could be private to the package)

Reply:
@yurishkuro implemented - can you take a look?